about-us.html

```diff
@@ -0,0 +1,543 @@
+                            </a>
+                        </li>
+                        <li class="our-team-socialmedia-item">
+                            <a href="#">
+                                <img src="./img/" alt="telegram">
+                            </a>
+                        </li>
+                    </ul>
+                </li>
+                <li class="team-item">
+                    <img src="./img/" alt="Robert Fox">
+                    <h3>Robert Fox</h3>
+                    <p>UI/UX Designer</p>
+                    <ul class="our-team-socialmedia-list">
+                        <li class="our-team-socialmedia-item">
+                            <a href="#">
+                                <img src="./img/" alt="linkedin">
+                            </a>
+                        </li>
+                        <li class="our-team-socialmedia-item">
+                            <a href="#">
+                                <img src="./img/" alt="facebook">
+                            </a>
+                        </li>
+                        <li class="our-team-socialmedia-item">
+                            <a href="#">
+                                <img src="./img/" alt="instagram">
+                            </a>
+                        </li>
+                        <li class="our-team-socialmedia-item">
+                            <a href="#">
+                                <img src="./img/" alt="telegram">
+                            </a>
+                        </li>
+                    </ul>
+                </li>
+                <li class="team-item">
+                    <img src="./img/" alt="Devon Lane">
+                    <h3>Devon Lane</h3>
+                    <p>Ux Architect</p>
+                    <ul class="our-team-socialmedia-list">
+                        <li class="our-team-socialmedia-item">
+                            <a href="#">
+                                <img src="./img/" alt="linkedin">
+                            </a>
+                        </li>
+                        <li class="our-team-socialmedia-item">
+                            <a href="#">
+                                <img src="./img/" alt="facebook">
+                            </a>
+                        </li>
+                        <li class="our-team-socialmedia-item">
+                            <a href="#">
+                                <img src="./img/" alt="instagram">
+                            </a>
+                        </li>
+                        <li class="our-team-socialmedia-item">
+                            <a href="#">
+                                <img src="./img/" alt="telegram">
+                            </a>
+                        </li>
+                    </ul>
+                </li>
+               </ul>
+            </div>
+        </section>
+        <section class="partners">
+            <div>
```

Comment:
This element is an unnecessary wrapper. It can be removed without affecting the layout. Avoid unnecessary elements in the markup.